### ISSUE TITLE
[BUG] Banning node - Losing Sync

### ISSUE THREAD

**Issue by racabu** [Chia-Network/chia-blockchain#4701]
2021-05-12T07:38:59.224 full_node full_node_server        : WARNING  Peer node-or.chia.net is still banned, not connecting to it
2021-05-12T07:38:59.225 full_node chia.rpc.rpc_server     : WARNING  Error while handling message: Traceback (most recent call last):
  File "chia\rpc\rpc_server.py", line 212, in safe_handle
  File "chia\rpc\rpc_server.py", line 200, in ws_api
  File "chia\rpc\rpc_server.py", line 161, in open_connection
ValueError: Start client failed, or server is not set

2021-05-12T08:01:23.179 full_node full_node_server        : WARNING  Banning node-or.chia.net for 600 seconds
2021-05-12T08:01:23.184 full_node chia.full_node.full_node: ERROR    Error with syncing: <class 'RuntimeError'>Traceback (most recent call last):
  File "chia\full_node\full_node.py", line 628, in _sync
RuntimeError: Weight proof did not arrive in time from peer: node-or.chia.net

**Comment by markusuk1** [comment]
i also have same issue

**Comment by fullet-tortuga** [comment]
I'm also having this error yet haven't been able to work around it. Disabling the firewall obviously does not solve it. I haven't found any blocklist either among the Chia files so I don't know why a node may get banned.

**Comment by arnoudius** [comment]
I'm currently experiencing the exact same issue

**Comment by 0frostmourne0** [comment]
same here

**Comment by suleymancetin** [comment]
same here also

**Comment by jakub-polak** [comment]
Same for me. @racabu were you able to fix it?

**Comment by Justin-chr** [comment]
Still, the same Problem here!

**Comment by racabu** [comment]
I was able to resolve this by changing the Timezone in windows to Automatic instead of manually configuring it.

**Comment by racabu** [comment]
I'm still losing sync from time to time.
I have a harvester in Linux and the Farmer on windows.

**Comment by PaperMan013** [comment]
Have this issue once in two-three day. Only harvester reboot helps. Stand alone farmer Debian 10, stand alone harvester Centos 7. Block-chain 1.20 both. Previous versions was the same. Problems started when the number of plots become more then 300. Now it is 750.

**Comment by ajr1303** [comment]
I was having this same exact issue.  Figured out why I kept having my harvester show as connected for a second and disconnected and over and over again.  No matter what i did it continue to do this.   My resolution was stupid and simple.  Make sure the same version of Chia is installed on every machine.  If you have version 1.2.2 on the Node and 1.1.7 on the harvester you will have this issue.  Not saying this will fix your issue but it fixed mine and I hope this helps.
